locales/vi-VN/translation.json

```diff
@@ -5,50 +5,50 @@
   "New wallet": "Tạo ví mới",
   "Import wallet": "Thêm ví đã có sẳn",
   "Amount must be greater than {{ minAmountInAlph }}": "Số lượng phải lớn hơn {{ minAmountInAlph }}",
-  "Could not connect to the {{ currentNetwork }} network.": "Không thể kết nối đến mạng {{ currentNetwork }}.",
-  "Note that if activated, \"{{ address }}\" will not be the default address anymore.": "Lưu ý, nếu đã kích hoạt, \"{{ address }}\" sẽ không còn là địa chỉ mặc định nữa.",
+  "Could not connect to the {{ currentNetwork }} network.": "Không thể kết nối đến Network hiện tại {{ currentNetwork }}.",
+  "Note that if activated, \"{{ address }}\" will not be the default address anymore.": "Lưu ý: Nếu kích hoạt, \"{{ address }}\" sẽ không còn là địa chỉ mặc định nữa.",
   "Error while sweeping address {{ from }}": "Lỗi khi sweep từ địa chỉ {{ from }}",
   "Remove wallet \"{{ walletName }}\"": "Gỡ bỏ ví \"{{ walletName }}\"",
   "Enter password for \"{{ switchToWalletName }}\"": "Nhập mật khẩu cho \"{{ switchToWalletName }}\"",
   "{{ amount }} words entered": "{{ amount }} từ đã thêm vào",
   "Address label": "Nhãn ví",
-  "The wallet is offline.": "Ví offline.",
+  "The wallet is offline.": "Ví đang offline.",
   "Discreet mode": "Ẩn số dư",
   "Default address": "Địa chỉ mặc định",
   "Settings": "Cài đặt",
   "Copied to clipboard!": "Đã sao chép vào clipboard!",
   "Copy to clipboard": "Sao chép vào clipboard",
   "Copied": "Đã sao chép",
-  "Official Wallet": "Ví chính thức",
-  "The easiest way to get started with Alephium.": "Cách đơn giản nhất để bắt đầu với Alephium.",
+  "Official Wallet": "Ví Desktop chính thức",
+  "The easiest way to get started with Alephium.": "Alephium là blockchain Layer 1 đầu tiên giúp mở rộng quy mô và cải thiện trên các công nghệ cốt lõi của Bitcoin, Proof of Less Work & UTXO.",
```

Comment:
I am quite sure this is not one-to-one translation of the original text

Reply:
- well, the last Vietnamese version was ambiguous meaning. Therefore, I proposed to add _Desktop_ word into in order to consolidate the context meaning. It means that this is "_The Official Desktop Wallet_", not any kind else. Anyway, if this is not an acceptable in translating unification of the team, please lmk. I will return the basic meaning before. 
- line 23, also the unclear meaning of pure-translation. I proposed to add this quote `Alephium is the first operational sharded L1 blockchain scaling and enhancing PoW & UTXO concepts.` for users easier to understand. I understand it change totally the original meaning of the english version in this line, but Vietnamese people cannot understand when I translated it as pure-translation.

Conclusion, please advice me for the direction. Thanks :-)